chain-core/src/init/config.rs

```diff
@@ -54,6 +55,9 @@ impl fmt::Display for DistributionError {
             DistributionError::InvalidVotingPower => {
                 write!(f, "Invalid voting power")
             },
+            DistributionError::InvalidPunishmentConfiguration => {
+                write!(f, "Invalid punishment configuration (maybe slash_wait_period < jail_duration)")
```

Comment:
This message seems to imply that the `slash_wait_period` should not be less than `jail_duration`; I think it should give some hinting words to describe you should make `slash_wait_period < jail_duration`

Reply:
Changed